enclave/key.c

```diff
@@ -323,7 +323,7 @@ oe_result_t oe_public_key_free(oe_public_key_t* public_key, uint64_t magic)
     if (public_key)
     {
         if (!oe_public_key_is_valid(public_key, magic))
-            OE_RAISE(OE_INVALID_PARAMETER);
+            OE_RAISE_NO_TRACE(OE_INVALID_PARAMETER);
```

Comment:
Again the rationale is not clear here. Why no trace?

Reply:
I was trying to remove unnecessary error messages: in some error cases , when jumping to "done" label, we ended up spewing confusing error message for those clean up operations.